### PR TITLE
Update README to reflect recent ClickHouse releases

### PR DESCRIPTION
**Pull request by onorua**
- Downsampling is implemented in v18.12.14
- Delta compression is implemented in v19.3.3

Fixes https://github.com/Percona-Lab/PromHouse/issues/26

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/Percona-Lab/PromHouse?pullRequest=27) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/Percona-Lab/PromHouse?pullRequest=27) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/Percona-Lab/PromHouse?pullRequest=27) it.</sub>

File: README.md [hunk 1]
```diff
@@ -24,8 +23,8 @@ ENGINE = ReplacingMergeTree
 
 CREATE TABLE samples (
     fingerprint UInt64,
-    timestamp_ms Int64,
-    value Float64
+    timestamp_ms Int64 CODEC(Delta),
+    value Float64 CODEC(Delta
```
Comment (leoluk) [inline README.md]: Missing `)`

Comment (onorua) [reply README.md]: good catch! thanks!

**Comment by AlekSi** [comment]
Oops, missed that one. Thanks!

**Pull request merged**